src/bake/DevServer.zig

```diff
@@ -7032,17 +7322,27 @@ pub const Assets = struct {
 
 /// Storage for source maps on `/_bun/client/{id}.js.map`
 ///
-/// All source maps are referenced counted, so that when a bundle is unloaded,
-/// the unreachable source map URLs are revoked, and their memory freed.
+/// All source maps are referenced counted, so that when a websocket disconnects
+/// or a bundle is replaced, the unreachable source map URLs are revoked. Source
+/// maps that aren't reachable from IncrementalGraph can still be reached by
+/// a browser tab if it has a callback to a previously loaded chunk; so DevServer
+/// should be aware of it.
 pub const SourceMapStore = struct {
-    entries: AutoArrayHashMapUnmanaged(u64, Entry),
+    pub const Key = bun.GenericIndex(u64, .{ "Key of", SourceMapStore });
+
+    entries: AutoArrayHashMapUnmanaged(Key, Entry),
+    /// Shared between all entries
+    weak_ref_sweep_timer: ?*uws.Timer,
```

Comment:
Don't use uws.Timer. We should delete it from the codebase. Use EventLoopTimer instead. EventLoopTimer avoids creating a file descriptor for each timer.
```suggestion
    weak_ref_sweep_timer: ?*uws.Timer,
```